docs/findings/el8.md

```diff
@@ -22,59 +22,63 @@
+  | `The OS Must Ensure Session Control Is Automatically Started At Shell Initialization`_     | V-230349            |
+  |                                                                                            |                     |
+  |                                                                                            | RHEL-08-020041      |
+  +--------------------------------------------------------------------------------------------+---------------------+
+  | `User Account Passwords Must Be Restricted To A 60-Day Maximum Lifetime`_                  | V-230367            |
+  |                                                                                            |                     |
+  |                                                                                            | RHEL-08-020210      |
+  +--------------------------------------------------------------------------------------------+---------------------+
+  | `OS Must Prohibit Password Reuse For A Minimum Of Five Generations`_                       | V-230368            |
+  |                                                                                            |                     |
+  |                                                                                            | RHEL-08-020220      |
+  +--------------------------------------------------------------------------------------------+---------------------+
+  | `The Installed Operating System Is Not Vendor Supported`_                                  | V-230221            |
+  |                                                                                            |                     |
+  |                                                                                            | RHEL-08-010000      |
+  +--------------------------------------------------------------------------------------------+---------------------+
+  | `All Content In A User's Home Directory Must Be Group-Owned By The Primary User`_          | V-244532            |
+  |                                                                                            |                     |
+  |                                                                                            | RHEL-08-010741      |
+  +--------------------------------------------------------------------------------------------+---------------------+
+  | `"Only Authorized Local User Accounts Exist on Operating System" is always flagged`_       | V-230379            |
+  |                                                                                            |                     |
+  |                                                                                            | RHEL-08-020320      |
+  +--------------------------------------------------------------------------------------------+---------------------+
+  | `All Interactive User Home Directory Files Must Have Be Mode 0750 Or Less Permissive`_     | V-244531            |
```

Comment:
"Must Have Be"? Is that right?

Reply:
Blargh. No. Was trying to make the text marginally more-compact and "missed".